source/platforms/GitLab.ts

```diff
@@ -92,6 +92,23 @@ class GitLab implements Platform {
     return true
   }
 
+  supportsThreads() {
+    return true;
+  }
+
+  updateOrCreateThread = async (dangerID: string, newComment: string): Promise<string> => {
+    const threads = await this.getDangerThreads(dangerID);
+
+    if (threads.length) {
+      await this.api.updateMergeRequestDiscussion(threads[0].id, threads[0].notes[0].id, newComment);
+      return `${this.api.mergeRequestURL}#note_${threads[0].notes[0].id}`;
+    } else {
+      const thread = await this.api.createMergeRequestDiscussion(newComment);
+
+      return `${this.api.mergeRequestURL}#note_${thread.notes[0].id}`;
```

Comment:
Maybe the zero could be a const with a description. Looks like a magic number without any type of context of why the code should the zero index and not any other